modules/adot/variables.tf

```diff
@@ -74,3 +102,27 @@ variable "adot_log_group_name" {
   type        = string
   default     = "adot_log_group_name"
 }
+
+# variable "resources" {
```

Comment:
why we need this commented block?
if no need just remove it, if we need it have some comment about this.